### DISCUSSION TITLE
Documentation for v4.6.1

### DISCUSSION THREAD

**Discussion by devpythonk** [miguelgrinberg/python-socketio#877] [Q&A] [answered]
hi,
I am working with a socket io server whose versions are 
Socket.IO version - 2.3.0
Engine.IO version - 3.4.0

the corresponding match for python-socketio was 4.x , so i was looking for docs of version 4.x but couldn't find any. Can you share steps on how to get that

**Comment by miguelgrinberg** [comment, marked as answer]
I'm not building documentation for older releases. You can build it locally. First install sphinx in your virtualenv:

    pip install sphinx

Then build the docs as follows:

    cd docs
    make html

The above requires `make` to be installed on your system. After the command ends, you can open the documentation in the `_build/html/index.html` under `docs`.
